### ISSUE TITLE
tab navigation: context menu opened via shortcut opens at weird position

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-desktop#4244]
- navigating via `Tab` to a message's three-dot-menu to open the context menu
- hit `Space` or `Enter` to open the menu

=> the menu is opened in the upper left corner of the screen; on mac even below the window control elements

(instead the context menu should be opened beside the three-dot-menu, or at least at a position not covered by buttons)

https://github.com/user-attachments/assets/a78d41d3-bfee-4380-a956-69a7aa33cb1e

tested on macOs14.5 with deltachat1.46.8

**Comment by WofWca** [comment]
Same goes for opening "react" through a message's context menu with keyboard.
